www/src/components/searchbar-body.js

```diff
@@ -0,0 +1,225 @@
+           }
+         }}
+       />
+     </div>
+
+     <div css={{
+       borderTop: `2px solid #F5F3F7`,
+       borderBottom: `2px solid #F5F3F7`,
+       borderLeft: `2px solid #F5F3F7`,
+     }}>
+
+        <div css={{
+          backgroundColor: `white`,
+          height: `calc(100vh - 225px)`,
+          border: `2 px solid red`,
+          overflowY: `scroll`,
+          WebkitOverflowScrolling: `touch`,
+          "::-webkit-scrollbar": {
+            width: `6px`,
+            height: `6px`,
+          },
+          "::-webkit-scrollbar-thumb": {
+            background: colors.ui.bright,
+          },
+          "::-webkit-scrollbar-track": {
+            background: colors.ui.light,
+          },
+        }}>
+          <Hits hitComponent={Result} />
+        </div>
+
+    </div>
+
+      <div>
+        <h3 css={{
+          fontSize: rhythm(.55),
+          textAlign: `center`,
+          margin: rhythm(.75),
+        }}>Search by <a href={`https://www.algolia.com/`} style={{color: `#744C9E`, border: `none`, boxShadow: `none`}}>Algolia</a></h3>
+      </div>
+
+    </div>
+  )
+}
+
+const Result = ({ hit }) => {
+  const selected = location.pathname.slice(10) === hit.name
+  const lastUpdated = `${distanceInWords(new Date(hit.modified), new Date())} ago`;
+  return (
+    <Link
+      to={`/packages/${hit.name}`}
+      style={{
+      display: `block`,
+      fontFamily: typography.options.bodyFontFamily.join(`,`),
+      fontWeight: `400`,
+      color: selected ? `white` : `black`,
+      backgroundColor: selected ? `#744C9E` : `white`,
+      padding: rhythm(.5),
+    }}>
+      <div css={{
+        display: `flex`,
+        justifyContent: `space-between`,
+      }}>
+        <div
+          css={{
+            fontFamily: typography.options.headerFontFamily.join(`,`),
+            fontWeight: `bold`,
+          }}
+          >
+            {hit.name}
+          </div>
+
+          <div
+            css={{
+              display: `flex`,
+              alignItems: `center`,
+              fontSize: rhythm(.5),
+            }}
+            >
+              {hit.humanDownloadsLast30Days}
+              <img width="10"
+                height="10"
+                css={{
+                  marginLeft: rhythm(.25),
+                  marginBottom: 0,
+                }}
+                src={selected ? DownloadsWhite : DownloadsBlack}></img>
+            </div>
+      </div>
+
+
+      <div css={{
+        fontSize: rhythm(.6),
+        paddingTop: rhythm(.25),
+        lineHeight: rhythm(.75),
+      }}>
+        {hit.description}
+      </div>
+
+      {/* <div css={{
+        fontSize: rhythm(.5),
+        color: `#D3D3D3`,
+      }}>
+        {hit.keywords.join(", ")}
+      </div> */}
+
+      {/* <div
+        css={{
+          display: `flex`,
+          paddingTop: rhythm(.25),
+        }}
+        >
+        <img
+          width="20"
+          height="20"
+          src={hit.lastPublisher.avatar}
```

Comment:
`hit.owner` instead of `lastPublisher`, it's usually more relevant than whoever published last